packages/common/src/projects/_internal/computeProps.ts

```diff
@@ -39,10 +39,7 @@ export function computeProps(
 
   project.view = {
     ...model.data.view,
-    featuredImageUrl: getAuthedImageUrl(
-      model.data.view?.featuredImageUrl,
-      requestOptions
-    ),
+    featuredImageUrl: model.data.view?.featuredImageUrl,
```

Comment:
same thing here. You can get rid of lines 40-43 since we aren't overwriting anything in `project.view`